combat-trainer.lic

```diff
@@ -142,7 +142,7 @@ end
 class LootProcess
   include DRC
 
-  def initialize(skin:false, arrange_all:false, arrange_count:0, lootables:[], arrange_types:{})
+  def initialize(skin:false, arrange_all:false, arrange_count:0, lootables:[], arrange_types:{}, ritual_type:preserve, necro:false)
```

Comment:
preserve isn't a valid default value here, it's not a string or anything else. this should probably be `ritual_type:{}` since an empty hash seems like the correct default for ritual type.